pymwp/semiring.py

```diff
@@ -27,28 +28,29 @@
 def prod_mwp(scalar1: str, scalar2: str) -> str:
     """Compute product of two scalars.
 
-    | $\\times$ | $0$ | $m$      | $w$       | $p$      | $\infty$ |
-    | ---       | --- | ---      | ---       | ---      | --- |
-    | $0$       | $0$ | $0$      | $0$       | $0$      | $0$ |
-    | $m$       | $0$ | $m$      | $w$       | $p$      | $\infty$ |
-    | $w$       | $0$ | $w$      | $w$       | $p$      | $\infty$ |
-    | $p$       | $0$ | $p$      | $p$       | $p$      | $\infty$ |
-    | $\infty$  | $0$ | $\infty$ | $\infty$  | $\infty$ | $\infty$ |
+    | $\\times$  | $0$ | $m$       | $w$        | $p$       | $\\infty$ |
+    | ---        | --- | ---       | ---        | ---       | --- |
+    | $0$        | $0$ | $0$       | $0$        | $0$       | $0$ |
```

Comment:
We need to have that last 0 becomes an \infty :-)
I know, this is a different issue…